### ISSUE TITLE
Support for Version Catalog plugins' bundles?

### ISSUE THREAD

**Issue by krzdabrowski** [gradle/gradle#20872]
I couldn't find similar issue so I'm posting a new one.

### Expected Behavior
I would like to use `[bundles]` in .toml Version Catalog not only for `[libraries]` items but also for `[plugins]`.

### Current Behavior
I'm getting the exception when trying to do it:
```
Exception is:
java.lang.RuntimeException: org.gradle.api.InvalidUserDataException: Invalid catalog definition:
  - Problem: In version catalog libs, a bundle with name 'exampleBundle' declares a dependency on 'examplePlugin' which doesn't exist.
    
    Reason: Bundles can only contain references to existing library aliases.
    
    Possible solutions:
      1. Make sure that the library alias 'examplePlugin' is declared.
      2. Remove 'examplePlugin' from bundle 'exampleBundle'.
```

### Context
It's quite similar as for dependencies, quoting docs:
> Because it’s frequent that some dependencies are systematically used together in different projects, a version catalog offers the concept of a "dependency bundle". 

In multi-module project, I often find useful to reuse some groups of plugins. Currently I need to define them one by one.

**Comment by MarvinSchramm** [comment]
Hej Hej ,i s there still a plan to assess the viability of this?
We are also very interested in this feature since our services also share a lot of plugins which we currently maintain manually.
🙏

**Comment by alllex** [comment]
Sorry for the late reply

This issue needs a decision from the team responsible for that area. They have been informed. Response time may vary.

***

As an alternative for extending the version catalog functionality, the [Declarative Gradle initiative](https://github.com/gradle/gradle/issues/28337) could address your request about bundling plugins in the form of _Software Types_.

**Comment by big-guy** [comment]
I don't think we want to support bundles for plugins.  We see a bundle of plugins that belong together as a conventions plugin that would apply each of them.

**Comment by sarimmehdi** [comment]
Hi @big-guy I don't understand your answer. If you have a lot of modules and all of them need the plugins related to hilt applied, wouldn't it just be good create a bundle for plugins and then put that inside the plugins block instead of writing down multiple lines of plugins every time?

**Comment by Hamed1370m** [comment]
0xabcdef13
